### PR TITLE
delete stuck pods in valid_namespaces

### PR DESCRIPTION
**Pull request by MUzairS15**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — DISMISSED** [review]
That’ll work! 🤠

**Comment by cpepper96** [comment]
So this will just list all the pods that are not currently running or being created right? @MUzairS15

**Comment by leecalcote** [comment]
> So this will just list all the pods that are not currently running or being created right? @MUzairS15

@cpepper96, good eye. I don't know that this PR takes us to conclusion with a deletion of unwanted Pods....

**Comment by MUzairS15** [comment]
> So this will just list all the pods that are not currently running or being created right? @MUzairS15

Yes

**Comment by leecalcote** [comment]
in that case, how is this helpful?

**Comment by MUzairS15** [comment]
The PR is for deleting stuck/crashing pods in namespaces we don’t delete (valid namespaces).

**Comment by MUzairS15** [comment]
For resources (or pods) in other namespaces we are deleting the entire namespace hence covered.

**Comment by cpepper96** [comment]
I feel like I'm missing something here :laughing: 

@MUzairS15 are you planning on adding to this script to delete the stuck pods or just list them? Sidenote, can we just wipe the whole `default` namespace each night? Is there anything being deployed there that we want to persist?

File: install/playground/playground-cleanup.sh [hunk 1]
```diff
@@ -19,6 +19,12 @@ do
     fi
 done
 
+for ns in $valid_namespaces;
+do
+    kubectl get pods -n $ns \
```
Comment (MUzairS15) [inline install/playground/playground-cleanup.sh:24]:
```suggestion
    kubectl delete pods -n $ns \
```

**Comment by MUzairS15** [comment]
> I feel like I'm missing something here 😆
> 
> @MUzairS15 are you planning on adding to this script to delete the stuck pods or just list them? Sidenote, can we just wipe the whole `default` namespace each night? Is there anything being deployed there that we want to persist?

Thanks for pointing out. I was testing if the command is able to select required pods correctly and later forgot to replace `get` with `delete`. 😅

**Comment by MUzairS15** [comment]
Yes we can but still there can be stale pods in other namespaces.

**Comment by leecalcote** [comment]
@MUzairS15 should this be marked as draft or work-in-progress?

**Comment by MUzairS15** [comment]
It is completed.

**Comment by MUzairS15** [comment]
It is completed

**Pull request merged**